File: app/src/main/java/com/websarva/wings/android/kaigonote/SyusinngoyakuActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.SyusinngoyakuBinding;

public class SyusinngoyakuActivity extends AppCompatActivity {
    private SyusinngoyakuBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = SyusinngoyakuBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/SyusinngoyakuActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.SyusinngoyakuBinding

class SyusinngoyakuActivity : AppCompatActivity() {
    private var binding: SyusinngoyakuBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = SyusinngoyakuBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}